docs/Contributing/API-for-contributors.md

```diff
@@ -1345,18 +1345,18 @@ If the `name` is not already associated with an existing team, this API route cr
         "macos_settings": {
           "custom_settings": {
             "path": "path/to/profile1.mobileconfig"
-            "labels": ["Label 1", "Label 2"]
+            "labels_include_all": ["Label 1", "Label 2"]
```

Comment:
Add a second name for existing `labels` key: `labels_include_all`

Both keys have the same behavior: profile is only applied to hosts that have all labels.